analyses/collapse-rnaseq/run-collapse-rnaseq.sh

```diff
@@ -19,17 +19,54 @@ cd "$script_directory" || exit
 # create results directory if it doesn't already exist
 mkdir -p results
 
+# Usage: project acronym to use as prefix for input out files 
+usage(){ echo "Usage: $0 [-h] [-p <project acronym>] [-l <library strategy>]" 1>&2; exit 1; }
+
+while getopts ":hl:p:" opt; do
+    case "${opt}" in
+	h)
+	    usage
+	    ;;
+	p)
+	    project_acronym=$OPTARG
+	    ;;
+	l)
+	    libraryStrategies=$OPTARG
+	    ;;
+	:)
+	    printf "missing argument for -%s\n" "$OPTARG" 1>&2
+	    usage
+	    ;;
+	\?)
+	    printf "illegal option: -%s\n" "$OPTARG" 1>&2
+	    usage
+	    ;;
+    esac
+done
+shift $((OPTIND - 1))
+
+if [ -z "${project_acronym}" ]; then
+    usage
+fi
+
+if [ -z "${libraryStrategies}" ]; then
+    usage
+fi
+
+
 # generate collapsed matrices for poly-A and stranded datasets
-libraryStrategies=("polya" "stranded")
+#libraryStrategies=("polya" "stranded")
+#libraryStrategies=("stranded")
```

Comment:
I think you can now remove these commented code:
```suggestion

```

Reply:
@kgaonkar6 Sounds good. I have removed the commented lines.